integration-tests/07_course.ts

```diff
@@ -21,6 +22,7 @@ const courseOne = test('Create Course One', async () => {
             description: "Why should I test if my users can do that for me in production?"
             category: club
             allowContact: true
+            shared: true
```

Comment:
I think you need a separate mutation markShared afterwards

Reply:
Can you please rephrase this comment? Do you mean that we need a separate test for shared and nonshared courses? For the current integration test I manually set the shared value to false. See line 414

Reply:
No the courseCreate Mutation does not take a shared attribute, thus the test is failing?